test/views/ContactCheckYourAnswersViewSpec.scala

```diff
@@ -325,5 +288,7 @@ object ContactCheckYourAnswersViewSpec {
     ContactInfo("name2", "role2", "email2", "phone2"),
     ContactInfo("name3", "role3", "email3", "phone3")
   )
+  val pageHeading         = "Check your answers"
+  val submitButtonContent = "Save and Continue"
```

Comment:
can we move these to the companion object of this spec please, since they are constants